packages/@o3r/eslint-plugin/src/index.ts

```diff
@@ -39,19 +39,34 @@ module.exports = {
       }
     },
 
+    // deprecated: should use `angular-template-recommended` instead
     'template-recommended': {
       rules: {
         '@o3r/no-folder-import-for-module': 'error',
         '@o3r/template-async-number-limitation': 'warn'
       }
     },
 
+    // deprecated: should use `monorepo-recommended` instead
     'json-recommended': {
       rules: {
         '@o3r/json-dependency-versions-harmonize': 'error'
       }
     },
 
+    'angular-template-recommended': {
```

Comment:
This config contains o3r rules 
why renaming it with `angular` prefix ?

Reply:
The `@o3r/` prefix remains, but this are recommended for angular template parsing only.
If you see another name, no problem to propose it :)

Reply:
Does the no-inner-html rely on angular ? 
the name doesn't seem to imply this

Reply:
It forbids `[innterHtml]` directive in an Angular Template.
(It also requires the `@angular-eslint/template-parser`)